components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/internal/DashboardMetadataProviderImpl.java

```diff
@@ -60,22 +65,28 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DashboardMetadataProviderImpl.class);
     private static final String PERMISSION_APP_NAME = "DASH";
+    private static final String VIEWER_PERMISSION_SUFFIX = ".viewer";
```

Comment:
shall we use `PERMISSION_SUFFIX_VIEWER` instead.
- `PERMISSION_SUFFIX_EDITOR`
- `PERMISSION_SUFFIX_OWNER`